web/stories/testgrid-index.stories.ts

```diff
@@ -19,6 +19,36 @@ interface ArgTypes {
   backgroundColor?: string;
 }
 
+// data
+const dashboards = [
+  'cert-manager-jetstack-testing-janitors',
+  'cert-manager-periodics-master',
+  'cert-manager-periodics-release-1.10',
+  'cert-manager-periodics-release-1.11',
+  'cert-manager-presubmits-master',
+  'conformance-all',
+  'conformance-apisnoop',
+  'conformance-cloud-provider-huaweicloud',
+  'gardener-ci-infra',
+  'gardener-dependency-watchdog',
+  'gardener-extension-networking-calico',
+  'google-aws',
+  'google-cel',
+  'google-gce',
+];
+
+const dashboardGroups = ['cert-manager', 'conformance', 'gardener', 'google'];
+
+const respectiveDashboards = {
+  'cert-manager': [
```

Comment:
It's best to use your own data instead of a snapshot of a production (if public) database for testing.

Reply:
Done!